substrate/client/executor/polkavm/src/lib.rs

```diff
@@ -250,12 +278,16 @@ where
 		},
 	};
 
-	let module = polkavm::Module::from_blob(&engine, &polkavm::ModuleConfig::default(), blob)?;
-	let mut linker = polkavm::Linker::new(&engine);
+	let module =
+		polkavm::Module::from_blob(&engine, &polkavm::ModuleConfig::default(), blob.clone())?;
```

Comment:
This clone doesn't seem to be necessary.

```suggestion
		polkavm::Module::from_blob(&engine, &polkavm::ModuleConfig::default(), blob)?;
```

Reply:
It must be a left-over from refactorizations. I'll apply this change.

Reply:
My bad. I did not realize you are getting a reference to a `ProgramBlob` here. So the `clone()` was needed.

Reply:
NP, reverted.